.travis.yml

```diff
@@ -19,8 +19,12 @@ install:
 script:
   - mix format --check-formatted
   - mix credo
+  - mix dialyzer --halt-exit-status
   - mix hex.outdated || true
   - mix test
+cache:
+  directories:
+    - priv/plts
```

Comment:
By default, PLTs are generated in `_build/dev`. This won't cache them correctly.

And we probably don't want to cache `_build/dev` because of all the other stuff that's in there.

Reply:
You can confirm this in your Travis output:

https://travis-ci.org/smartcitiesdata/divo/builds/569596897#L606